src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+Example .. code-block:: html 
+
+            <ad:engines>
+                <engine>promotions</engine>
+            </ad:engines>
+======= =======================================
+
+.. index:: microTip
+.. _feed_microTip:
+
+microTip
+""""""""
+
+A short freeform text with a maximum length of 18 characters, excluding any characters in ``.,/@#<>``.
+It is a feature as part of a package that sellers can purchase (currently available only for Marktplaats tenant).
+It provides extra attention on the ad in the search results.
+
+======= ===========================================
+Example .. code-block:: html
+        
+         <ad:microTip>TODAY 15% SALE</ad:microTip>
+======= ===========================================
+
+.. index:: mpn
+.. _feed_mpn: 
+
+MPN
+"""
+
+Manufacturer Part Number (MPN), definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+String identifier max 2-70 chars.
+
+======= ==================================
+Example .. code-block:: html
+        
+         <ad:mpn>AB12345R89TN6E</ad:mpn>
+======= ==================================
+
+.. index:: googleProductCategory
+.. _feed_googleProductCategory: 
+
+googleProductCategory
+""""""""""""""""""""""
+
+Product category from Google's product taxonomy. See `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_
+
+======= =============================================================================================================
+Example .. code-block:: html
+        
+         <ad:googleProductCategory>
+            Apparel &amp; Accessories &gt; Clothing &gt; Dresses
+         </ad:googleProductCategory>
+======= =============================================================================================================
+
+.. index:: productType
+.. _feed_productType: 
+
+productType
+""""""""""""""""""""""
+
+Definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+String identifier max 750 chars.
+
+======= ===============================================================================
+Example .. code-block:: html
+        
+         <ad:productType>
+            Home &gt; Women &gt; Dresses &gt; Maxi Dresses
+         </ad:productType>
+======= ===============================================================================
+
+.. index:: brand
+.. _feed_brand: 
+
+brand
+""""""""""""""""""""""
+
+Brand definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
+String identifier max 70 chars.
+
+======= ============================================
+Example .. code-block:: html
+        
+         <ad:brand>Little black dress</ad:brand>
+======= ============================================
+
+.. index:: gtin
+.. _feed_gtin: 
+
+GTIN
+""""""""""""""""""""""
+
+GTIN, definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
```

Comment:
```suggestion
GTIN (Your product’s Global Trade Item Number ), definition follows `Google Merchant Center <https://support.google.com/merchants/answer/7052112>`_ guidelines.
```

Reply:
it might just spark interest in those who are not tied to 3rd party vendors like channable, but manually create feeds